app/jobs/pageflow/reset_counter_caches.rb

```diff
@@ -0,0 +1,13 @@
+module Pageflow
+  class ResetCounterCachesJob
+    @queue = :default
+
+    extend StateMachineJob
+
+    def self.perform_with_result(file, options = {})
```

Comment:
Unused method argument - `file`. If it's necessary, use `_` or `_file` as an argument name to indicate that it won't be used. You can also write as `perform_with_result(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `perform_with_result(*)` if you want the method to accept any arguments but don't care about them.